homeassistant/components/alarm_control_panel/__init__.py

```diff
@@ -155,6 +160,19 @@ class AlarmControlPanelEntity(Entity, cached_properties=CACHED_PROPERTIES_WITH_A
     _attr_supported_features: AlarmControlPanelEntityFeature = (
         AlarmControlPanelEntityFeature(0)
     )
+    _alarm_control_panel_option_default_code: str = ""
+
+    @final
+    @callback
+    def add_default_code(self, code: str | None) -> str | None:
+        """Add default lock code."""
```

Comment:
```suggestion
        """Add default arm code."""
```

I'm not sure if lock makes sense here